src/App.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import Home from './pages/Home';
+import Coins from './pages/Coins';
+import Detail from './pages/Detail';
+import Navbar from './components/Navbar';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <Navbar />
+      <Routes>
+        <Route index element={<Home />} />
+        <Route path="/Coins" element={<Coins />} />
```

Comment:
- [ ] Your route for all the items is the same, `/Coins`. But they should be unique.

  For example, if we click an item on the homepage, we should have something like this: `/coins/<id for the item>`. This is a hint of what the route should look like.

  ```
  /coins/:id
  ```
  Kindly refactor your routes so that all the coins have their own unique routes. You can consider looking at **`useParams`** from the `react-router-dom` package, and using the URL id to filter or fetch the coin